pyfstat/make_sfts.py

```diff
@@ -370,9 +373,33 @@ def run_makefakedata(self):
         cl_mfd.append("--outSingleSFT=TRUE")
         cl_mfd.append('--outSFTdir="{}"'.format(self.outdir))
         cl_mfd.append('--outLabel="{}"'.format(self.label))
-        cl_mfd.append("--IFOs={}".format(self.IFOs))
+
         if self.add_noise:
+
+            if self.noiseSFTs is not None and self.SFTWindowType is None:
+                raise ValueError(
+                    "SFTWindowType is required when using noiseSFTs. "
+                    "Please, make sure you understand the window function used "
+                    "to produced noiseSFTs."
+                )
+            elif self.noiseSFTs is not None:
+                cl_mfd.append('--noiseSFTs="{}"'.format(self.noiseSFTs))
+            else:
+                cl_mfd.append("--IFOs={}".format(self.IFOs))
+
+            logging.warning(
+                "Gaussian noise *is added* by default due to "
+                "consistency issues. Please, make sure sqrtSX value is "
+                "consistent with what you intend to do (0=No Gaussian noise). "
+                "Currently, sqrtSX={}".format(self.sqrtSX)
```

Comment:
That whole situation may require some further thought, I'm not so sure if the separate `add_noise` flag really makes sense either. But for now it's a good idea to put in that warning and if I end up unhappy with the sqrtSX situation, I'll make a separate issue/PR. Thanks.

Reply:
I'm not sure how many things will break down when removing `add_noise`. Agree with doing a separate issue/PR.

Reply:
see #50